src/HereAuth/HereAuth.php

```diff
@@ -221,7 +221,7 @@ public function onDisable(){
 	public function startUser(Player $player){
 		$this->database->loadFor($player->getName(), $player->getId());
 		if($player->spawned){
-			$player->sendMessage("[HereAuth] Your account data are being loaded. Please wait patiently; it shouldn't take long.");
+			$player->sendMessage("[HereAuth] Your account data is being loaded. Please wait patiently; it shouldn't take long.");
```

Comment:
@ifvictr you say "Your pants **are** on fire", not "Your pants **is** on fire", even though you are just referring to all the pants in your pair of pants.